src/package.ts

```diff
@@ -92,6 +92,38 @@ export class Package extends AsyncOptionalCreatable {
     return versions.includes(this.nextVersion);
   }
 
+  public writePackageJson(rootDir?: string): void {
+    const pkgJsonPath = rootDir ? path.join(rootDir, 'package.json') : 'package.json';
+    fs.writeJsonSync(pkgJsonPath, this.projectJson);
+  }
+
+  public hardcodeDependencyVersions(tag = 'latest'): ProjectJson {
+    const hardcodedDependencies: NpmPackage[] = [];
+    // get the list of dependencies to hardcode
+    const dependencies: string[] = this.projectJson['packagesToHardcodeVersions'];
+    dependencies.forEach((name) => {
+      // get the 'release' tag version or the version specified by the passed in tag
+      const result = exec(`npm view ${name} dist-tags --json`, { silent: true });
+      const versions = JSON.parse(result.stdout);
+      // if tag is 'latest-rc' and there's no latest-rc release for a package, default to latest
+      let version;
+      if (!versions[tag]) {
+        process.emitWarning(`Warning: no tag ${tag} for package: ${name} - default to 'latest'`);
```

Comment:
This seems a little strong. Also it seems useful to see in the output all the things that have changes. What if this method returns 

```
type ChangedPackageVersions = Array<{
   name: string,
   version: string,
   tag: string
}>
```

That the way run method could print out something like:
```
Changed package versions:

name                     | version | tag
------------------------ | ------- | ------
salesforcedx             |  1.2.3  | latest-rc
@salesforce/plugin-apex  |  1.2.3  | latest
@salesforce/plugin-auth  |  1.2.3  | latest
```